src/main/java/com/suse/saltstack/netapi/datatypes/Token.java

```diff
@@ -31,4 +34,8 @@ public Date getStart() {
     public Date getExpire() {
         return expire;
     }
+
+    public List<String> getPerms() {
```

Comment:
That's right, @moio, let's keep the original name. I will actually cherry-pick  4139a4a and a8833da then (= everything until the renaming).